### PR TITLE
Added types and functions required by the GX29 app

### PR DESCRIPTION
**Pull request by mcrispino**
* Added ChooseImage & TakePhoto implementations that delegate the task to the generator
* Added Image.IsEmpty() method
* Added SetEmpty() method for character types (Char, Varchar, LongVarchar & Image)
* Added Interop.iOSSetBadgeTextToTabIndex() method
* Added support for GeneXus.Common.UI.Navigation external object that delegates the tasks to the generator, including the ShowTarget method.
* Added Numeric.ToFormattedString as an alias of Numeric.ToString
* Added support for PlaceCall & SendEmail methods from Interop

### CODE REVIEW

File: src/gxcore/sd/interop/canOpen.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if an URL can be opened
+ * Current implementation only returns `true` for http and https URLs
+ * @param url
+ */
+export const canOpen = (url: string): boolean => {
+  return (
```
Comment (ncardeli) [inline src/gxcore/sd/interop/canOpen.ts]: How about relative URLs?
For example:
* //example.com
* /about-us
* ../index.html
* about-us/contact

File: src/text/setEmpty.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Assigns the empty value in an attribute or variable
+ * @param target
+ * @return string
+ */
+
+export const setEmpty = (target: string): string => {
+  target = "";
```
Comment (ncardeli) [inline src/text/setEmpty.ts]: How about just returning an empty string?

File: src/gxcore/sd/interop/canOpen.ts (hunk 1 above)
Comment (mcrispino) [reply src/gxcore/sd/interop/canOpen.ts]: What do you suggest? :)

File: src/text/setEmpty.ts (hunk 2 above)
Comment (mcrispino) [reply src/text/setEmpty.ts]: Fixed at (7499d27)[https://github.com/genexuslabs/web-standard-functions/pull/57/commits/7499d27ab5ae41628072ff4a9dae201d97a2a6b5]

File: src/gxcore/sd/interop/canOpen.ts (hunk 1 above)
Comment (ncardeli) [reply src/gxcore/sd/interop/canOpen.ts]: Maybe we could use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL)

Comment (mcrispino) [reply src/gxcore/sd/interop/canOpen.ts]: Made some changes at [43869aa](https://github.com/genexuslabs/web-standard-functions/pull/57/commits/43869aa3f3e753fd264bb23930e122a67199505b), please take a look.

**Pull request merged**